src/Server/modules/DataClasses.js

```diff
@@ -94,7 +94,7 @@ module.exports = {
 	},
 
 	WorldPlace: class {
-		constructor(country, state, latitude, longitude) {
+		constructor(country, state, latitude, longitude, currentNumConfirmed, currentNumDeaths, currentNumRecovered) {
```

Comment:
If we add these variables to the argument, we also need to modify modules/ReadCSV.js LoadWorldData(). 

Instead, these three variables can be populated inside addNumDeaths, addNumConfirmed, and addNumRecovered when the dictionaries are populated.

Reply:
oh sorry i added it thinking it would fix some error but i forgot to take it out. I'll take it out